src/index.html

```diff
@@ -8,6 +8,71 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+    </ul>
+    <ul class="stars stars--1">
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+    </ul>
+    <ul class="stars stars--2">
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
```

Comment:
```suggestion
    <ul class="stars stars--0">
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
    </ul>
    
    <ul class="stars stars--1">
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
    </ul>
    
    <ul class="stars stars--2">
      <li class="stars__star"></li>
      <li class="stars__star"></li>
```
Don't forget to add newlines between blocks of code to improve readability. Also, you can remove newlines between simple elements of block